lnwallet/interface.go

```diff
@@ -80,6 +80,19 @@ var ErrNoOutputs = errors.New("no outputs")
 var ErrInvalidMinconf = errors.New("minimum number of confirmations must " +
 	"be a non-negative number")
 
+// ErrMempoolFee is returned from PublishTransaction in case the tx being
+// published is not accepted into mempool because the fee requirements of the
+// mempool backend are not met. In addition it allows to wrap the error message
+// from the backend.
+type ErrMempoolFee struct {
+	Message string
```

Comment:
Looks like this `Message` field is not used?

Reply:
https://github.com/lightningnetwork/lnd/pull/7746/files#diff-e542f325bd2d7e4c20e498088a5794249b117f46faf31aacd8ab6ebda9c6e37cR1219 we basically wrap the original error.